zebrad/src/components/tracing/component.rs

```diff
@@ -32,6 +47,9 @@ impl Tracing {
         let filter = config.filter.unwrap_or_else(|| "".to_string());
         let flame_root = &config.flamegraph;
 
+        // By default, the built NonBlocking will be lossy. (with a line limit of 128_000)
```

Comment:
What happens when the line limit is reached?
Can you document that here?

Reply:
It ignores any new calls to write instead of queuing them in the channel, I added some more detail here